stl/inc/chrono

```diff
@@ -5461,9 +5471,10 @@ struct formatter<_CHRONO sys_time<_Duration>, _CharT> {
 };
 */
 
-template <class _CharT>
-struct formatter<_CHRONO day, _CharT> {
-    typename basic_format_parse_context<_CharT>::iterator parse(basic_format_parse_context<_CharT>& _Parse_ctx) {
+template <class _CharT, bool _Allow_precision>
+struct _Chrono_formatter {
+    typename basic_format_parse_context<_CharT>::iterator _Parse(
+        basic_format_parse_context<_CharT>& _Parse_ctx, string_view _Legal_types) {

```

Comment:
Nitpick: Recommend renaming this to `_Valid_types`, we generally don't say "legal". This will also match the exception messages that talk about "invalid types".